site/layouts/shortcodes/svg-docs.html

```diff
@@ -1,48 +1,24 @@
 {{- /*
-  Usage: `svg-docs file="img/orange-logo.svg" downloadable="true"`
+  Usage: `svg-docs file="site/static/docs/{version}/assets/brand/orange-logo" downloadable="true"`
 
     Prints SVG content.
 
     Optional parameters:
-      file: SVG file path - default: `img/orange-logo.svg`
+      file: SVG file path without file extension - default: `site/static/docs/{version}/assets/brand/orange-logo`
       downloadable: is the file downloadable? - default: `true`
 */ -}}
 
-{{- $file := .Get "file"| default "img/orange-logo.svg" -}}
-{{- $svg := readFile $file -}}
-{{- $code := $svg -}}
+{{- $file := .Get "file" | default (print "site/static/docs/" .Site.Params.docs_version "/assets/brand/orange-logo") -}}
```

Comment:
Going with last last comment
```suggestion
{{- $name := .Get "file" | default "orange-logo" -}}
{{- $file := (print "site/static/docs/" .Site.Params.docs_version "/assets/brand/" $name) -}}
```
Or something like this could avoid us applying the `5.2` version and keep it automated